check/configuration/supported/terraform.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Terraform CLI
+---
+
+# Terraform
+
+Terraform is a formatter for infra.
```

Comment:
Terraform is a framework so this line is misleading/weird. Need to work on this one's copy